protocols/bsbport/bsbport.h

```diff
@@ -46,21 +46,21 @@ struct bsbport_msg
 
 struct bsbport_buffer_msg
 {
-  uint16_t act;
+  uint8_t act;
```

Comment:
It is not ensured that the buffer size in protocols/bsbport/config.in does not exceed 256. Also applies to similar changes in this file.